llama-index-integrations/readers/llama-index-readers-google/llama_index/readers/google/drive/base.py

```diff
@@ -20,22 +22,55 @@
 SCOPES = ["https://www.googleapis.com/auth/drive.readonly"]
 
 
-class GoogleDriveReader(BaseReader):
-    """Google drive reader."""
+class GoogleDriveReader(BasePydanticReader):
```

Comment:
We missed adding the staticmethod `def class_name()` method -- it will default to a base name, which is not helpful for deserializing

Reply:
my b— didn't realize that's how this worked. will go through and add these in

Reply:
my b for not catching this either, :)